bigtable/client/client_options_test.cc

```diff
@@ -112,7 +112,7 @@ TEST(ClientOptionsTest, EditCredentials) {
 
 TEST(ClientOptionsTest, SetGrpclbFallbackTimeout) {
   bigtable::ClientOptions client_options_object = bigtable::ClientOptions();
-  client_options_object.SetGrpclbFallbackTimeout(5);
+  client_options_object.SetGrpclbFallbackTimeout(std::chrono::milliseconds(5));
```

Comment:
Please add a new unit test that uses `std::chrono::seconds()` too.

Reply:
Shall I add unit test for std::out_of_range exception also?

Reply:
Good point, yes, please do.